validator/rpc/structs.go

```diff
@@ -89,3 +89,46 @@ type GetFeeRecipientByPubkeyResponse struct {
 type SetFeeRecipientByPubkeyRequest struct {
 	Ethaddress string `json:"ethaddress"`
 }
+
+// Type of key manager for the wallet, either derivedKeymanagerKind, importedKeymanagerKind, or remote.
+type KeymanagerKind string
+
+const (
+	derivedKeymanagerKind    KeymanagerKind = "DERIVED"
+	importedKeymanagerKind   KeymanagerKind = "IMPORTED"
+	web3signerKeymanagerKind KeymanagerKind = "WEB3SIGNER"
```

Comment:
I think repeating the type name in the name of the const is a code smell. I would just call it `derived/ etc

Reply:
I think it clashed with something that's why I specified

Reply:
Ah, probably with a package name or something. OK in that case let's leave it